### PR TITLE
feat(numeric-menu): implement InstantSearch.css

### PR DESCRIPTION
**Pull request by francoischalifour**
## Specs

https://instantsearch-css.netlify.com/widgets/numeric-menu/

## Stories

https://deploy-preview-3162--algolia-instantsearch.netlify.com/v2/dev-novel/?selectedStory=NumericMenu.default

### CODE REVIEW

File: dev/app/init-unmount-widgets.js [hunk 1]
```diff
@@ -179,10 +179,10 @@ export default () => {
...
   );
 
-  storiesOf('NumericRefinementList').add(
+  storiesOf('NumericMenu').add(
     'default',
     wrapWithUnmount(container =>
-      instantsearch.widgets.numericRefinementList({
+      instantsearch.widgets.numericMenu({
         container,
         attributeName: 'price',
```
Comment (samouss) [inline dev/app/init-unmount-widgets.js]: `attribute`

File: dev/app/jquery/widgets/numericMenu.js [hunk 2]
```diff
@@ -5,11 +5,7 @@ const renderFn = (
   {
     items,
     refine,
-    widgetParams: {
-      containerNode,
-      attributeName,
-      title = 'Numeric refinement list',
-    },
+    widgetParams: { containerNode, attributeName, title = 'Numeric menu' },
```
Comment (samouss) [inline dev/app/jquery/widgets/numericMenu.js]: `attribute`

File: src/widgets/numeric-menu/numeric-menu.js [hunk 3]
```diff
@@ -116,10 +116,7 @@ numericMenu({
  *       {end: 500, name: 'less than 500'},
```
Comment (samouss) [inline src/widgets/numeric-menu/numeric-menu.js]: Should be `label`

File: src/widgets/numeric-menu/defaultTemplates.js [hunk 4]
```diff
@@ -1,8 +1,7 @@
 /* eslint-disable max-len */
 export default {
-  header: '',
   item: `<label class="{{cssClasses.label}}">
-  <input type="radio" class="{{cssClasses.radio}}" name="{{attributeName}}" {{#isRefined}}checked{{/isRefined}} />{{label}}
+  <input type="radio" class="{{cssClasses.radio}}" name="{{attribute}}" {{#isRefined}}checked{{/isRefined}} />
```
Comment (samouss) [inline src/widgets/numeric-menu/defaultTemplates.js]: The `attribute` value is not provided to the template so we still have `undefined`.

```html
<input type="radio" class="ais-NumericMenu-radio" name="">
```

File: src/widgets/numeric-menu/numeric-menu.js [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+ */
+
+/**
+ * @typedef {Object} NumericMenuTemplates
+ * @property {string|function} [item] Item template, provided with `label` (the name in the configuration), `isRefined`, `url`, `value` (the setting for the filter) data properties.
+ */
+
+/**
+ * @typedef {Object} NumericMenuOption
+ * @property {string} name Name of the option.
```
Comment (samouss) [inline src/widgets/numeric-menu/numeric-menu.js]: Should be `label`

File: dev/app/builtin/stories/numeric-menu.stories.js [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+export default () => {
+  stories
+    .add(
+      'default',
+      wrapWithHits(container => {
+        window.search.addWidget(
+          instantsearch.widgets.numericMenu({
+            container,
+            attribute: 'price',
+            operator: 'or',
```
Comment (Haroenv) [inline dev/app/builtin/stories/numeric-menu.stories.js]: how does `or` make sense on a menu? A menu can only have one refinement, so can't have an operator (I think?) 🤔

Comment (francoischalifour) [reply dev/app/builtin/stories/numeric-menu.stories.js]: Right, that probably was there before. Removing it.

File: src/components/RefinementList/__tests__/__snapshots__/RefinementList-test.js.snap [hunk 7]
```diff
@@ -47,6 +48,7 @@ exports[`RefinementList rendering with facets 1`] = `
         dangerouslySetInnerHTML={
           Object {
             "__html": Object {
+              "attribute": "attribute",
```
Comment (Haroenv) [inline src/components/RefinementList/__tests__/__snapshots__/RefinementList-test.js.snap]: is this correct?

Comment (francoischalifour) [reply src/components/RefinementList/__tests__/__snapshots__/RefinementList-test.js.snap]: Yes, we now pass the `attribute` too to the template to fill the `name` HTML property:

https://github.com/algolia/instantsearch.js/blob/feat/3.0-numeric-menu/src/components/RefinementList/RefinementList.js#L48

https://github.com/algolia/instantsearch.js/blob/feat/3.0-numeric-menu/src/widgets/numeric-menu/defaultTemplates.js#L4

**Pull request merged**